packages/graph/hash_graph/lib/graph/src/store/postgres/query/expression/mod.rs

```diff
@@ -0,0 +1,13 @@
+mod conditional;
+mod join;
+mod select;
+mod r#where;
+mod with;
+
+pub use self::{
+    conditional::{Expression, Function},
+    join::JoinExpression,
+    select::SelectExpression,
+    r#where::WhereExpression,
```

Comment:
I'm personally okay with this module name as we wouldn't call SQL `where` something else. @Alfred-Mountfield what do you think? We could call it `filter`, or somethign else, but that would break the 1:1 correspondance

Reply:
Wow this is horrible 😁 

Honestly I'm not sure what I'd do, I don't have a strong feeling either way, it's just unfortunate it's a protected keyword.